app/helpers/application_helper.rb

```diff
@@ -597,4 +597,11 @@ def concurrent_events(event)
     end
     concurrent_events
   end
+
+  def speaker_selector_input(form)
+    users = User.active.pluck(:id, :name, :username, :email).map { |user| [user[0], user[1].blank? ? user[2] : user[1], user[2], user[3]] }.sort_by { |user| user[1] }
```

Comment:
Also this sorts capitalized words first, and non-capitalized words later, which does not produce a truly sorted list